test/utils/windowUtilsTests.ts

```diff
@@ -1,8 +1,52 @@
 ///<reference path="../testReference.ts" />
 
 describe("Utils.Window", () => {
-  describe("deprecated()", () => {
+  describe("warn()", () => {
+    const runTest = console != null ? it : it.skip;
```

Comment:
I am unsure if this is appropriate.  I currently perceive tests to be behavior to be tested later (Tests that are declared but unimplemented).  (This is more highlighted in that Mocha will mark these tests as pending).

As a result, we should be aiming to have as few pending tests as possible.  In situations like these, I am more inclined to complete the test immediately if `console` is not defined.